src/transformers/models/fastspeech2/configuration_fastspeech2.py

```diff
@@ -0,0 +1,154 @@
+        speaker_embed_dim=64,
+        dropout=0.2,
+        max_source_positions=1024,
+        encoder_attention_heads=2,
+        fft_hidden_dim=1024,
+        fft_kernel_size=9,
+        attention_dropout=0,
+        encoder_layers=4,
+        decoder_embed_dim=256,
+        decoder_attention_heads=2,
+        decoder_layers=4,
+        add_postnet=False,
+        postnet_conv_dim=512,
+        postnet_conv_kernel_size=5,
+        postnet_layers=5,
+        postnet_dropout=0.5,
+        vocab_size=75,
+        num_speakers=1,
+        var_pred_n_bins=256,
+        var_pred_hidden_dim=256,
+        var_pred_kernel_size=3,
+        var_pred_dropout=0.5,
+        pitch_max=5.733940816898645,
+        pitch_min=-4.660287183665281,
+        energy_max=3.2244551181793213,
+        energy_min=-4.9544901847839355,
+        initializer_range=0.0625,
+        mean=True,
+        std=True,
+        pad_token_id=1,
+        **kwargs
+    ):
+        if fft_kernel_size % 2 == 0:
+            raise ValueError(f"`fft_kernel_size` must be odd, but got {fft_kernel_size} instead.")
+        if postnet_conv_kernel_size % 2 == 0:
+            raise ValueError(f"`postnet_conv_kernel_size` must be odd, but got {postnet_conv_kernel_size} instead.")
+        if var_pred_kernel_size % 2 == 0:
+            raise ValueError(f"`var_pred_kernel_size` must be odd, but got {var_pred_kernel_size} instead.")
+        self.n_frames_per_step = n_frames_per_step
+        self.output_frame_dim = output_frame_dim
+        self.encoder_embed_dim = encoder_embed_dim
+        self.speaker_embed_dim = speaker_embed_dim
+        self.dropout = dropout
+        self.max_source_positions = max_source_positions
+        self.encoder_attention_heads = encoder_attention_heads
+        self.fft_hidden_dim = fft_hidden_dim
+        self.fft_kernel_size = fft_kernel_size
+        self.attention_dropout = attention_dropout
+        self.encoder_layers = encoder_layers
+        self.decoder_embed_dim = decoder_embed_dim
+        self.decoder_attention_heads = decoder_attention_heads
+        self.decoder_layers = decoder_layers
+        self.add_postnet = add_postnet
+        self.postnet_conv_dim = postnet_conv_dim
+        self.postnet_conv_kernel_size = postnet_conv_kernel_size
+        self.postnet_layers = postnet_layers
+        self.postnet_dropout = postnet_dropout
+        self.vocab_size = vocab_size
+        self.num_speakers = num_speakers
+        self.var_pred_n_bins = var_pred_n_bins
+        self.var_pred_hidden_dim = var_pred_hidden_dim
+        self.var_pred_kernel_size = var_pred_kernel_size
+        self.var_pred_dropout = var_pred_dropout
+        self.pitch_min = pitch_min
+        self.pitch_max = pitch_max
+        self.energy_min = energy_min
+        self.energy_max = energy_max
+        self.initializer_range = initializer_range
+        self.mean = mean
+        self.std = std
+        super().__init__(pad_token_id=pad_token_id, **kwargs)
```

Comment:
can we move this above all the attributes? :-)

Reply:
Yes, that makes sense. I'll move it right below the `ValueError` checks!